modules/common/log/logs-process.nix

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  cfg = config.ghaf.logging.client;
+  listenerAddress = "admin-vm-debug";
+  listenerPort = config.ghaf.logging.listener.port;
+  # TODO: replace sshCommand and MacCommand with givc rpc to retrieve Mac Address
+  sshCommand = "${pkgs.sshpass}/bin/sshpass -p ghaf ${pkgs.openssh}/bin/ssh -o StrictHostKeyChecking=no ghaf@net-vm";
+  MacCommand = "cat /sys/class/net/wlp0s5f0/address";
+  remoteLoggingAddress = config.ghaf.logging.server.address;
+  loggingPort = config.ghaf.logging.port;
+  lokiForward =
+    if config.ghaf.logging.server.enable
+    then "loki.write.local.receiver, loki.write.remote.receiver"
+    else "loki.write.local.receiver";
+in {
+  config = lib.mkIf cfg.enable {
+    # TODO: Remove hw-mac.service and replace with givc rpc later
+    systemd.services."hw-mac" = {
```

Comment:
Can this be moved to another module, even with the hacky workaround we have currently? All that alloy has to do with it is read from /var/lib/private/alloy/MACAddress, but it doesn't need to be coupled to actually also contain the implementation of that.

Reply:
Good point. Decoupled and moved different module.